lib/portus/jwt_token.rb

```diff
@@ -5,6 +5,8 @@ module Portus
   #   https://github.com/docker/distribution/blob/master/docs/spec/auth/token.md
   #
   class JwtToken
+    DEFAULT_EXPIRATION_TIME = 5.minutes
```

Comment:
This seems to be used only when there's no `jwt_expiration_time` value configured inside of the `config.yml`. That should not happen right?
Also this is used when the value is something wrong; in that case I would just let everything crash. That would make the issue evident.